### ISSUE TITLE
Order History not dimming all cancelled trades

### ISSUE THREAD

**Issue by fliebenberg** [DeXter-on-Radix/website#485]
In my Order History tab, not all the cancelled trades are showing as dimmed.
![image](https://github.com/user-attachments/assets/ef18b1a1-6c2f-4347-8d0a-9ac7eb464090)

**Comment by dcts** [comment]
@maurojose can you give your input on this issue?

**Comment by maurojose** [comment]
true, It's confusing. As order history shows only orders that are not open, I think we could dim every cancelled order, and maybe put a button or checkbox like "hide unfilled orders"

**Comment by fliebenberg** [comment]
So I suggest we go with dimming all cancelled orders for now to avoid confusion. Alternatively we could have 2 levels of dimming, with only half dimming of partially filled orders. That might work, although I dont know if you will actually notice the difference.
